### ISSUE TITLE
val_loss is increasing

### ISSUE THREAD

**Issue by huangcong77** [fizyr/keras-retinanet#155]
I'm training the network on VOC 2007. While the loss is decreasing , the val_regression_loss stay the same (0.0022) and the val_classification_loss is increasing after epoch 5（from 0.3348 to 0.8527）. Is  this model over-fitting?

**Comment by hgaiser** [comment]
Yes sounds like it. Though I've also noticed that a higher loss doesn't necessarily mean worse performance. It only takes one anchor to properly detect an object. If other nearby anchors are classified as background it will result in a large loss but a high accuracy.
